sandbox/src/mocks/createReferral/requests/MinimalRequest.json

```diff
@@ -55,7 +55,7 @@
               {
                 "name": "specialty",
                 "valueCoding": {
-                  "system": "_baseUrl_/STU3/v1/CodeSystem/SPECIALTY",
+                  "system": "_baseUrl_/STU3/CodeSystem/SPECIALTY",
```

Comment:
These endpoints should also include "FHIR" I think, "_baseUrl_/FHIR/STU3/CodeSystem/SPECIALTY"

Reply:
As above, the base URL in the main spec has had the FHIR appended